CHANGELOG.md

```diff
@@ -8,8 +8,22 @@ This project follows [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) fo
 
 ## [Unreleased]
 
+## [1.5.0] - 2021-03-30
+
+* **Tracing**: Removed deprecated attributes on `Tracing` annotation. If you are still using `captureResponse` and `captureError` 
```

Comment:
Isn't that a major update?

What's the cost in keeping both but leaving only one documented?

Reply:
That's how we deprecated it initially by removing its reference from our docs. So docs don't have these attributes referenced. It won't lead to any surprises functionally, since this will lead to a compile time error for users. 

Cost of keeping deprecated attribute forever is basically technical debt.